prereq_flowchart/majorScrapper.py

```diff
@@ -0,0 +1,109 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+import json
+import re
+import time
+
+class Course:
+    credits: int
+    title:str
+    catalogTitle:str
+    description:str
+    inOr:bool #This is to indicate that the course has options
+    required:bool
+    def __init__(self,credits,title,catalogTitle,description,inOr,required):
+        self.credits=credits
+        self.title=title
+        self.catalogTitle=catalogTitle
+        self.description=description
+        self.inOr=inOr
+        self.required=required
+    def getJSON(self):
+        return{  "credits"      :self.credits,
+                 "title"        :self.title,
+                 "catalogTitle" :self.catalogTitle,
+                 "description"  :self.description,
+                 "inOr"         :self.inOr,
+                 "required"     :self.required}
```

Comment:
could this be a dataclass?

```suggestion
from dataclasses import dataclass

@dataclass
class Course:
    credits: int
    title:str
    catalogTitle:str
    description:str
    inOr:bool #This is to indicate that the course has options
    required:bool

    def getJSON(self):
        return{  "credits"      :self.credits,
                 "title"        :self.title,
                 "catalogTitle" :self.catalogTitle,
                 "description"  :self.description,
                 "inOr"         :self.inOr,
                 "required"     :self.required}
```